Java-Data-Types.md

```diff
@@ -0,0 +1,148 @@
+# Java data-Types
+
+Java is a strongly ­typed language. This means that, in Java, each data type has its own strict definition. There are no implicit data type conversions when any conflicts occur between the data types. Any change in data types should be explicitly declared by the programmer.
+ 
+Java defines 8 simple data types. They are divided into the following categories:
+ 
+Integers
+ 
+Floating­ Point Numbers
+ 
+Characters
+ 
+Boolean Type
+ 
+The details of each of the data types is given below
+ 
+### Integers:
+
+These are of four types: byte, short, int, long. It is important to note that these are signed positive and negative values. Java does not support positive­ only numbers.
+
+#### byte:
+
+    Byte data type is an 8-bit signed two's complement integer.
+
+    Minimum value: -128 (-2^7)
+
+    Maximum value: 127 (inclusive)(2^7 -1)
+
+    Default value: 0
+
+    Example: byte a = 10 , byte b = -500
```

Comment:
I think you just said that minimum value cannot be less than -128. What happens if you assign -500 to a `byte` variable?